src/manager.js

```diff
@@ -701,19 +715,37 @@ AdManager.prototype.refreshSlots = function (slotsToLoad) {
     return;
   }
 
+  var useIAS = typeof this.__iasPET !== 'undefined' && this.options.iasEnabled;
   var useIndex = typeof window.headertag !== 'undefined' && window.headertag.apiReady === true;
   var usePrebid = typeof window.pbjs !== 'undefined' && this.options.prebidEnabled;
+  var refreshMethod = function(){
```

Comment:
I think it makes sense for this to be defined on the `AdManager.prototype` so it can be independently unit tested